zict/common.py

```diff
@@ -23,16 +21,16 @@ def update(*args, **kwds):
         if args:
             other = args[0]
             if isinstance(other, Mapping) or hasattr(other, "items"):
-                items += other.items()
+                items = other.items()
```

Comment:
This is more of a general comment, but `items` is no longer a list, but in our [`check_items`](https://github.com/dask/zict/blob/6850845b645aea71bac342db9cafc8ed9546db4d/zict/tests/utils_test.py#L28-L35) used in [`check_mappings`](https://github.com/dask/zict/blob/6850845b645aea71bac342db9cafc8ed9546db4d/zict/tests/utils_test.py#L69-L113) in `utils_test.py`  we compare against a list. This still works but should we modify this, to be consistent?

Reply:
Not sure if I understand. check_items has nothing to do with the update() method?

Reply:
I agree that it hasn't but I noticed that we do things like this `check_items(z, [("abc", b"456"), ("xyz", b"12")])` where `check_items` takes z and does `list(z.item())`  to compare. I was wondering if we should compare the `z.items()` directly to the `dict_items()` but I guess it's not necessary.

Reply:
It's a test and it's not testing memory descoping.